app/routes/conversations/conversation.js

```diff
@@ -2,9 +2,8 @@ import { get } from '@ember/object';
 import { inject as service } from '@ember/service';
 import Route from '@ember/routing/route';
 import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
-import { CanMixin } from 'ember-can';
 
-export default Route.extend(AuthenticatedRouteMixin, CanMixin, {
```

Comment:
Seems `can` is just a template helper now and we now longer need the mixin for the route, but I'm not 100% sure.